src/app/clusters/resource-monitoring-server/resource-monitoring-server.cpp

```diff
@@ -197,6 +197,12 @@ Status Instance::OnResetCondition()
         }
     }
 
+    ReplacementProductListManager * productListManagerInstance = Instance::GetReplacementProductListManagerInstance();
+    if (nullptr != productListManagerInstance)
+    {
+        productListManagerInstance->Reset();
```

Comment:
Why?  This Reset means "we will now be starting at the beginning of the list".... but it's only relevant for reads of the attribute.  Why are we doing it here.

Reply:
Removed